accounts/urls.py

```
@@ -0,0 +1,7 @@
+from django.urls import path, include
+
+from . import views
```

Ruff (0.8.2)

`.views` imported but unused

Remove unused import: `.views`

(F401)